Subroto

Subroto-21 opened the pull request.

subroto123 committed:
Jokes api getting joke and converting it into array

subroto123 committed:
Merge branch 'main' of https://github.com/Shriteen/Joke-invaders into subroto

subroto123 committed:
Added start button

subroto123 committed:
Added start button

Shriteen committed:
Remove merge conflict markers

Shriteen committed:
Refinements to start screen

- Rename start to start-screen and moved it before game
- Remove inline styling and handle it fully via js using jquery
- CSS improvements

Shriteen commented:
I've made some changes to the code. I think its better to work on the filtering of joke filtering mechanism before merging with main.

Shriteen committed:
Filtering of jokes

- Jokes are now buffered rather than one at a time
- Ignore big jokes
- Replace problematic characters

Shriteen committed:
Delete jokes.js

Remove unused file
